src/types/sasjsconfig-schema.json

```diff
@@ -520,6 +528,34 @@
         }
       }
     },
+    "testConfig": {
+      "testFolders": ["sasjs/tests"],
+      "testJobFolders": ["sasjs/tests/jobs"],
+      "testServiceFolders": ["sasjs/tests/services"],
```

Comment:
we should have `testXxxFolders` **or** `testFolders` but not both

Reply:
my preference would be the `testXxxFolders`

Reply:
removed both from test configuration